src/util.rs

```diff
@@ -0,0 +1,47 @@
+use std::collections::HashMap;
+
+use matrix_sdk::{
+    ruma::{OwnedRoomAliasId, OwnedRoomId},
+    Client,
+};
+
+pub(super) struct Util {
```

Comment:
How about we call this `RoomResolver`, since it does just that at the moment? In the future if we want other utility functions, we can create other bespoke structures